test_common/harness/testHarness.c

```diff
@@ -945,3 +947,53 @@ Version get_device_cl_version(cl_device_id device)
 
     throw std::runtime_error(std::string("Unknown OpenCL version: ") + str.data());
 }
+
+void PrintArch( void )
+{
+    vlog( "sizeof( void*) = %ld\n", sizeof( void *) );
+#if defined( __ppc__ )
+    vlog( "ARCH:\tppc\n" );
+#elif defined( __ppc64__ )   
+    vlog( "ARCH:\tppc64\n" );
+#elif defined( __PPC__ )   
+    vlog( "ARCH:\tppc\n" );
+#elif defined( __i386__ )   
+    vlog( "ARCH:\ti386\n" );
+#elif defined( __x86_64__ )   
+    vlog( "ARCH:\tx86_64\n" );
+#elif defined( __arm__ )   
+    vlog( "ARCH:\tarm\n" );
+// Add 64 bit support
```

Comment:
Tiny nit-pick, but this comment could be removed.

Reply:
> Tiny nit-pick, but this comment could be removed.

Done